### PR TITLE
Add browser flag to testing

### PR DESCRIPTION
**Pull request by infinitepr0**
The reason this has been proposed is as follows:

- Many times, I may not wish to download the latest version of chromium, due to bandwidth restrictions.
- I may wish to test the library on a particular version of the browser of my choosing.

For these reasons, the PR provides a flag to specify while testing:
```bash
go test --browser-bin=<command to launch browser>
```
to set the browser to use to run tests.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -39,6 +39,15 @@ rod=show,trace,slow=2s go test -v -run /Click
 
 Check type `defaults.ResetWithEnv` for how it works.
 
+### Specify browser binary
+
+To use a locally installed browser, instead of downloading the latest version of chromium every time while running tests, use the `--browser-bin=<command to launch browser>` flag.
```
Comment (ysmood) [inline .github/CONTRIBUTING.md:44]:
```suggestion
You can use the `-browser-bin` flag to specify a custom browser executable path:
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -39,6 +39,15 @@ rod=show,trace,slow=2s go test -v -run /Click
...
+
+For example:
+```bash
+go test -v --browser-bin=google-chrome
```
Comment (ysmood) [inline .github/CONTRIBUTING.md:48]:
```suggestion
go test -v --browser-bin=/path/to/browser
```

Not just chrome, user can choose edge

File: setup_test.go [hunk 3]
```diff
@@ -83,9 +84,8 @@ func newTesterPool(t *testing.T) TesterPool {
 
 // new tester
 func (cp TesterPool) new() *T {
-	bin := ""
-
-	if !(utils.FileExists("/.dockerenv") || utils.FileExists("/.containerenv")) {
+	bin := *BrowserBin
+	if !(bin != "" || utils.FileExists("/.dockerenv") || utils.FileExists("/.containerenv")) {
```
Comment (ysmood) [inline setup_test.go:88]:
```suggestion
	const notInContainer = !(utils.FileExists("/.dockerenv") || utils.FileExists("/.containerenv"))
	if bin == "" && notInContainer {
```

**Review by ysmood — CHANGES_REQUESTED** [review]

**Comment by infinitepr0** [comment]
I can't do the linting part, since
1. I absolutely hate node.js
2. I'm getting the following error on trying: 
```
2020/12/25 12:45:25 setup project...
node: error while loading shared libraries: libicui18n.so.68: cannot open shared object file: No such file or directory
panic: exit status 127

goroutine 1 [running]:
github.com/go-rod/rod/lib/utils.E(...)
        /home/infinitepro/GitHub/infinitepr0/rod/lib/utils/utils.go:63
github.com/go-rod/rod/lib/utils.Exec(0x534ab3, 0x3, 0xc00008dee0, 0x6, 0x6)
        /home/infinitepro/GitHub/infinitepr0/rod/lib/utils/utils.go:341 +0x165
main.nodejsDeps()
        /home/infinitepro/GitHub/infinitepr0/rod/lib/utils/setup/main.go:25 +0x132
main.main()
        /home/infinitepro/GitHub/infinitepr0/rod/lib/utils/setup/main.go:13 +0x65
exit status 2
browser.go:1: running "go": exit status 1
```

Can you please do it for me? Thanks!

**Comment by ysmood** [comment]
Sure, let me handle it.

**Comment by infinitepr0** [comment]
> Sure, let me handle it.

It's fine, I handled it (had to full upgrade my system to get the lib files to work). Please review it when free!

**Review by ysmood — APPROVED** [review]

**Pull request merged**